src/providers/Poolprovider.js

```diff
@@ -114,39 +114,43 @@ export function PoolProvider({ children }) {
       tokens.push(x.poolToken)
     }
   }
-  useEffect(() => {
-    const loadBalanceData = async () => {
-      const tulipD = await tulipData.wallet.simplyTokenBalances({
-        user_address: account,
-        network: 'rinkeby',
-        tokens,
-        web3: {
-          eth: new Providers.Web3Provider(window.ethereum),
-        },
-      })
-      setBalance(tulipD)
-    }
-    const loadDepositData = async () => {
-      const deposits = []
-      const tulipF = await tulipData.farm.deposits({
-        user_address: account,
+  const loadBalanceData = async () => {
+    const tulipD = await tulipData.wallet.simplyTokenBalances({
+      user_address: account,
+      network: 'rinkeby',
```

Comment:
shouldn't be this network: rinkeby as a constant somewhere?

Reply:
removed hardcoded value